fxa/tests/test_oauth.py

```diff
@@ -661,36 +665,32 @@ def test_good_jwt_token(self):
     def test_wrong_key_jwt_token(self):
         self.verify_will_succeed = False
         bad_key = self.get_file_contents("bad-key.json")
-        result = jwt.encode({}, bad_key, "RS256", {"typ": "at+jwt"})
-        try:
-            self.client.verify_token(result)
-        except Exception:
-            return
-        raise Exception("verifying the token signed with the wrong key \
-                         should have caused an error.")
+        token = self._make_jwt({}, bad_key)
+        with self.assertRaises(fxa.errors.TrustError):
+            self.client.verify_token(token)
+        for c in responses.calls:
+            if c.request.url == 'https://server/v1/verify':
+                raise Exception("testing with a well-formed token with invalid signature \
+                                 should not have resulted in a call to /verify, but it did.")
 
     @responses.activate
     def test_expired_jwt_token(self):
         private_key = self.get_file_contents("private-key.json")
-        result = jwt.encode({"qwer": "asdf", "exp": 0}, private_key, "RS256", {"typ": "at+jwt"})
-        try:
-            self.client.verify_token(result)
-        except Exception:
-            return
-        raise Exception("verifying an expired token should have caused an error.")
+        token = self._make_jwt({"qwer": "asdf", "exp": 0}, private_key)
+        with self.assertRaises(fxa.errors.TrustError):
```

Comment:
I suddenly remembered the existence of `assertRaises` which captures the pattern we want here.